spec.html

```diff
@@ -5216,23 +5216,21 @@ <h1>
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It converts _argument_ to one of 2<sup>8</sup> integral Number values in the inclusive interval from *+0*<sub>𝔽</sub> to *255*<sub>𝔽</sub>.</dd>
+        <dd>It clamps _argument_ to one of 2<sup>8</sup> integral Number values in the inclusive interval from *+0*<sub>𝔽</sub> to *255*<sub>𝔽</sub>.</dd>
       </dl>
       <emu-alg>
         1. Let _number_ be ? ToNumber(_argument_).
-        1. If _number_ is *+∞*<sub>𝔽</sub>, return *255*<sub>𝔽</sub>.
-        1. If _number_ is *-∞*<sub>𝔽</sub>, return *+0*<sub>𝔽</sub>.
         1. If _number_ is *NaN*, return *+0*<sub>𝔽</sub>.
-        1. If ℝ(_number_) ≤ 0, return *+0*<sub>𝔽</sub>.
-        1. If ℝ(_number_) ≥ 255, return *255*<sub>𝔽</sub>.
-        1. Let _f_ be floor(ℝ(_number_)).
-        1. If _f_ + 0.5 &lt; ℝ(_number_), return 𝔽(_f_ + 1).
-        1. If ℝ(_number_) &lt; _f_ + 0.5, return 𝔽(_f_).
-        1. If _f_ is odd, return 𝔽(_f_ + 1).
-        1. Return 𝔽(_f_).
+        1. Let _mv_ be the extended mathematical value of _number_.
+        1. If _mv_ ≤ 0, return *+0*<sub>𝔽</sub>.
+        1. If _mv_ ≥ 255, return *255*<sub>𝔽</sub>.
```

Comment:
We should use https://tc39.es/ecma262/#clamping

Reply:
Hmm, would something like `Let _clamped_ be the result of clamping _mv_ between 0 and 255.` really be an improvement over these early returns if it's followed by steps to implement rounding?

Reply:
I mean, you could also early return for integers. But yes, I think the clamping is an improvement over an upper bound comparison and a lower bound comparison.

Reply:
Okay, done.